vcpkg/maintainers/functions/vcpkg_make_configure.md

```diff
@@ -0,0 +1,149 @@
+---
+title: vcpkg_make_configure
+description: Use vcpkg_make_configure to configure a Make-based project.
+ms.date: 01/30/2024
+---
+# vcpkg_make_configure
+
+Configure a Make-based project.
+
+## Usage
+        
+```cmake
+vcpkg_make_configure(
+    [AUTOCONFIG]
+    [COPY_SOURCE]
+    [DISABLE_MSVC_WRAPPERS]
+    [NO_CPPFLAGS]
+    [ADD_BIN_TO_PATH]
+    [NO_DEFAULT_OPTIONS]
+    [NO_MSVC_FLAG_ESCAPING]
+    [USE_RESPONSE_FILES]
+    [SOURCE_PATH <source-path>]
+    [OPTIONS <configure-setting>...]
+    [OPTIONS_RELEASE <configure-setting>...]
+    [OPTIONS_DEBUG <configure-setting>...]
+    [PRE_CONFIGURE_CMAKE_COMMANDS <cmake-command>...]
+    [LANGUAGES <language>...]
```

Comment:
I miss `SKIP_CONFIGURE`, or more generally options overriding the configure command. Is this postponed to a future revision?

Reply:
postponed until something uses it. No usage no interface.

Reply:
Well, it was broken until a few months ago. The workaround was installing no-op `configure`s.
Or wrappers for the actual configure scripts, as for openssl.

Reply:
I mean we would need `SKIP_CONFIGURE` if we could set `CONFIGURE_COMMAND`.